src/platform/backends/qemu/qemu_virtual_machine.h

```diff
@@ -23,6 +23,7 @@
 #include <shared/base_virtual_machine.h>
 
 #include <multipass/process/process.h>
+#include <multipass/ssh/openssh_key_provider.h>
```

Comment:
I think you only need `ssh_key_provider.h` here now, right?

Reply:
Right. It seems both includes work though as `OpenSshKeyProvider` is the concrete class that gets used.